src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -979,7 +979,7 @@ public boolean requiresWorkspaceForPolling() {
     private static class CheckOutTask extends UpdateTask implements FileCallable<List<External>> {
         private final UpdateTask task;
 
-        public CheckOutTask(Run<?, ?> build, SubversionSCM parent, ModuleLocation location, Date timestamp,
+        CheckOutTask(Run<?, ?> build, SubversionSCM parent, ModuleLocation location, Date timestamp,
```

Comment:
Unrelated I guess.

Reply:
just wanted to make it explicit, because the method signature has also changed